### DISCUSSION TITLE
Did development stop for this project?

### DISCUSSION THREAD

**Discussion by canedje** [matt8707/ha-fusion#425] [Q&A]
Nothing is happening anymore for a month now.
Is the project development stopped?
That should be a pitty

**Comment by mucki12** [comment]
@matt8707 
Can you confirm that HA Fusion is being further developed?

**Reply by al31c0** [reply]
@matt8707 is back😎

**Comment by Bad3m3ist3r** [comment]
would also like to have more updates / changes. I love the project and it would be a shame if it didn't continue. The fact that it is not progressing quickly may also be due to the fact that only one developer is working on it. However, it would be really nice to know what the status is here!

**Comment by matt8707** [comment]
No.
